starters/blog/package.json

```diff
@@ -51,13 +51,12 @@
     "url": "git+https://github.com/gatsbyjs/gatsby-starter-blog.git"
   },
   "scripts": {
-    "dev": "gatsby develop",
-    "lint": "eslint --ext .js,.jsx --ignore-pattern public .",
-    "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\"",
-    "format": "prettier --trailing-comma es5 --no-semi --single-quote --write 'src/**/*.js' 'src/**/*.md'",
+    "build": "gatsby build",
     "develop": "gatsby develop",
+    "fix-semi": "eslint --quiet --ignore-pattern node_modules --ignore-pattern public --parser babel-eslint --no-eslintrc --rule '{\"semi\": [2, \"never\"], \"no-extra-semi\": [2]}' --fix gatsby-node.js",
```

Comment:
Can we take this opportunity to standardize on the scripts, as well?

I don't really love including `eslint` like this in this starter--so perhaps we remove some of the cruft in this starter, as well as things like `.travis.yml`.

Minimally, I think every starter should have the following scripts defined:

- `develop`
- `build`
- `serve`
- `start` (alias for develop)
- `test` (pre-configured like we have below)

Reply:
(I'm not sold on `prettier` or the format script; what do you think?)

Reply:
Yeah, sure, why not. Starter can be scary with eslint :) I'l make changes soon.